modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -216,28 +216,6 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 			},
 			connectiontypes.ErrConnectionNotFound,
 		},
-		{
-			"current upgrade not found even though channel is in INITUPGRADE",
-			func() {
-				// crossing hellos
-				err := path.EndpointB.ChanUpgradeInit()
-				suite.Require().NoError(err)
-
-				suite.chainB.DeleteKey(host.ChannelUpgradeKey(path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID))
-			},
-			types.ErrUpgradeNotFound,
-		},
-		{
-			"proposed upgrade fields must be equal to existing upgrade fields in crossing hellos",
```

Comment:
in crossing hellos if the upgrade exists we now just take the existing upgrade fields, which are then validated by `checkUpgradeForCompatibility`